docs/developers/README.md

```diff
@@ -3,9 +3,9 @@ title: Developers
 sidebar_position: 1
 ---
 
-# Welcome to the Developers doc
+# Welcome to the Developer docs
```

Comment:
its called Developers doc in titles, sections etc so some may find it inconsistent.
I think it is fine though.

Reply:
I believe this wording is still better here, so resolving.